lib/syskit/models/port.rb

```diff
@@ -105,19 +105,14 @@ def to_component_port
             # @raise [SelfConnection]
             def connect_to(in_port, policy = {})
                 out_port = to_component_port
+                if out_port.respond_to?(:init_policy) &&
+                    [true, false].include?(out_port.init_policy)
+                    policy = policy.merge(init: out_port.init_policy)
+                end
```

Comment:
This should not be done here at all. Just do it
- in data reader creation
- in the system network generation